packages/fuselage-ui-kit/src/elements/CheckboxElement.tsx

```diff
@@ -0,0 +1,47 @@
+import { CheckBox, Box } from '@rocket.chat/fuselage';
+import * as UiKit from '@rocket.chat/ui-kit';
+import { type ReactElement } from 'react';
+
+import { useUiKitState } from '../hooks/useUiKitState';
+import type { BlockProps } from '../utils/BlockProps';
+
+type CheckboxElementProps = BlockProps<UiKit.CheckboxElement>;
+
+const CheckboxElement = ({
+  block,
+  context,
+  surfaceRenderer,
+}: CheckboxElementProps): ReactElement => {
+  const [{ loading }, action] = useUiKitState(block, context);
```

Comment:
Shouldn't be used the value here to be controlled? look at MultiStaticSelectElement as an example...